### PR TITLE
Update jupyter_notebook_config.py

### PR DESCRIPTION
**Pull request by theely**
Adding the capability to set jupyter port via the PORT environment variable.
This is especially useful if this Docker container runs on clusters managers like Mesos, DCOS , etc.

### CODE REVIEW

File: tensorflow/tools/docker/jupyter_notebook_config.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 from IPython.lib import passwd
 
 c.NotebookApp.ip = '*'
-c.NotebookApp.port = 8888
+c.NotebookApp.port = int(os.getenv('PORT', '') or 8888)
```
Comment (craigcitro) [inline tensorflow/tools/docker/jupyter_notebook_config.py]: You could even do `int(os.getenv('PORT', 8888))` if you want.

**Comment by craigcitro** [comment]
LGTM

**Comment by danmane** [comment]
@tensorflow-jenkins test this please

File: tensorflow/tools/docker/jupyter_notebook_config.py (hunk 1 above)
Comment (theely) [inline tensorflow/tools/docker/jupyter_notebook_config.py]: @craigcitro would you like me to change it to int(os.getenv('PORT', 8888))? I don't have a strong preference, actually your proposal seems cleaner to me.

Comment (craigcitro) [inline tensorflow/tools/docker/jupyter_notebook_config.py]: yeah, let's go ahead and change it.

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

**Pull request merged**